README.md

```diff
@@ -22,20 +22,32 @@ On Ubuntu simply install from the standard repositories:
 
 `sudo apt-get install virtualenvwrapper`
 
-Mac OSX can install `virtualenvwrapper` from brew:
+Mac OSX users can install `virtualenvwrapper` with brew:
 
 `brew install virtualenvwrapper`
 
+Installing with Antigen
+-----------------------
 
-Installation from Antigen
--------------------------
-
-Installing from [Antigen](https://github.com/zsh-users/antigen) is super easy! Just add the following line to your `.zshrc`:
+Installing with [Antigen](https://github.com/zsh-users/antigen) is super easy! Just add the following line to your `.zshrc`:
 
 ```
 antigen bundle MichaelAquilina/zsh-autoswitch-virtualenv
 ```
-if you want to set a default virtual environment then you can also export `AUTOSWITCH_DEFAULTENV` in
+
+Installing with Zgen
+--------------------
+
+Installing with [Zgen](https://github.com/tarjoilija/zgen) is also easy! Add the following line to your `.zshrc`:
+
+```zgen load unixorn/tumult.plugin.zsh```
```

Comment:
Just noticed this. This seems like its the incorrect path. Wouldn't
`zgen load MichaelAquilina/zsh-auto-switch-virtualenv` be correct?